base/applications/fontview/fontview.c

```diff
@@ -93,6 +93,27 @@ ErrorMsgBox(HWND hParent, DWORD dwMessageId, ...)
 	LocalFree(hMemText);
 }
 
+static void
+SuccessMsgBox(HWND hParent, DWORD dwMessageId, ...)
```

Comment:
This function could have been merged with the previous one, by adding an extra parameter to that other one for specifying the MB_ICONERROR or MB_ICONINFORMATION.

Reply:
fix: add overall FormatMsgBox proc